workspaces/liferay-ticket-workspace/readme.markdown

```diff
@@ -0,0 +1,24 @@
+# Liferay Ticket Workspace
+
+## Deployment
+
+First we have to enable the below feature flag:
+
+    # Enabling attachments for object entries
```

Comment:
Why is this using `#` wouldn't that render an `h1`, which would be nested inside a `h2`. That would be semantically wrong. Also no need to indent this.